### PR TITLE
Aio image push

### LINKED ISSUE #469: AIO images are not versioned with containerized releases
#### What would you like to be added:
AIO images should be cut as part of the release process and written to quay.io/microshift/microshift-aio.

#### Why is this needed:
AIO images are currently stored in the quay.io/microshift/microshift repo without a standardized tag format.  By pushing AIO images to a bespoke repo and keeping them in step with deployment releases, we'll improve the uxp for the app developer use case.

### PR DESCRIPTION
**Pull request by copejon**
**Which issue(s) this PR addresses**:

Closes #469 

This PR adds a process for building, tagging, and releasing multi-arch AIO container images to quay.io/microshift/microshift-aio.

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -144,49 +146,68 @@ stage_release_image_binaries() {
...
+  local image_tags="$2"
   # use docker cli directly for clarity, as this is a docker-only func
-  docker manifest create "$IMAGE_REPO:$VERSION" "${RELEASE_IMAGE_TAGS[@]}" >&2
-  docker tag "$IMAGE_REPO:$VERSION" "$IMAGE_REPO:latest"
-  docker manifest push "$IMAGE_REPO:$VERSION"
-  docker manifest push "$IMAGE_REPO:latest"
+  docker manifest create "$dest_repo:$VERSION" "${image_tags[*]}" >&2
+  docker tag "$dest_repo:$VERSION" "$dest_repo:latest"
+  docker manifest push "$dest_repo:$VERSION"
+  docker manifest push "$dest_repo:latest"
```
Comment (copejon) [inline scripts/release.sh:193]: These functions are made generic to support standalone and AIO build processes.

File: scripts/release.sh [hunk 2]
```diff
@@ -237,14 +258,24 @@ QUAY_OWNER=${QUAY_OWNER:="microshift"}
...
+push_container_manifest "$IMAGE_REPO" "${RELEASE_IMAGE_TAGS[@]}"          || exit 1
+
+# publish binaries 
+UPLOAD_URL="$(git_create_release "$API_DATA" "$TOKEN")"                   || exit 1
+git_post_artifacts "$STAGE_DIR" "$UPLOAD_URL" "$TOKEN"                    || exit 1
+
+# publish aio container 
+build_aio_container_images_artifacts                                      || exit 1
+push_aio_container_image_artifacts                                        || exit 1
+push_container_manifest "$AIO_IMAGE_REPO" "${AIO_RELEASE_IMAGE_TAGS[@]}"  || exit 1
```
Comment (copejon) [inline scripts/release.sh:281]: The only changes here are to space out the existing steps into logical groupings, without mutating the order, and to add the AIO steps last.  Added parameter passing to functions that were made generic above.

**Comment by mangelajo** [comment]
you had a tiny conflict, it's resolved now. I only wonder how long will it take to create both images, the problem is that we build the arm64 image compiling microshift under qemu-static-arm64, it'd be far more efficient to just build the microshift-arm64 binary, and then have a dockerfile that copied it inside (similar to packaging/images/components)

**Comment by cooktheryan** [comment]
/LGTM

merging

**Pull request merged**